### DISCUSSION TITLE
Add multiple views at once via admin API

### DISCUSSION THREAD

**Discussion by WLefever-Cegeka** [Informatievlaanderen/VSDS-LDESServer4J#729] [Ideas]
This is currently not supported, but should be an easy extension.

**Comment by Yalz** [comment]
This is possible when defining one collection. You could then add multiple views to it

**Reply by xdxxxdx** [reply]
Hello @Yalz ,

Also have question on this point, might be a good idea maybe add an example of this to the doc of LDES Server or the API ?

Thanks
